robot_skills/src/robot_skills/arm/arms.py

```diff
@@ -420,18 +346,20 @@ def collect_gripper_types(self, gripper_type):
                     self._has_specific_gripper_types(GripperTypes.PARALLEL))
         return self._has_specific_gripper_types(gripper_type)
 
-    @staticmethod
-    def _has_specific_gripper_types(gripper_type):
+    def _has_specific_gripper_types(self, gripper_type):
         """
         Verify whether the arm as the given type of specific gripper.
 
         :param gripper_type: Type of gripper to check for. Must not be a pseudo gripper type.
         :return: Gripper types that match the requirement.
         """
-        # TODO: Extend arm to have knowledge about the gripper type that it has.
-        if gripper_type == GripperTypes.PINCH:
-            return [GripperTypes.PINCH]
-        elif gripper_type == GripperTypes.PARALLEL:
+        if not hasattr(self, 'gripper'):
+            rospy.loginfo("This arm does not have a 'gripper' ")
```

Comment:
No, you could for instance have an arm without a gripper. And if you want to know whether it has a certain gripper type the answer should just be 'no'

Reply:
You are correct Lars, this should throw an error because the rest of this function relies on having a gripper. However both this function and collect gripper types should probably be moved somewhere else. Which I will leave for another PR

Reply:
If you make an issue I think that is fine!